packages/components/atoms/f-button/src/components/Button.vue

```diff
@@ -79,16 +79,25 @@ export default {
         },
         /**
          * Renders `Link` component if a `href` attribute is applied to the component
-         * Renders `Action` component if no `href` attribute is applied to the component
+         * Renders `RouterLink` component if a `to` attribute is applied to the component, avoids page reload compared to Link with `href`
+         * Renders `Action` component if no `href` attrivute is applied to the component
          */
         componentType () {
-            return this.$attrs.href ? 'link-button' : 'action-button';
+            if (this.$attrs.href) {
+                return 'link-button';
+            }
+
+            if (this.$attrs.to) {
+                return 'router-link';
+            }
+
+            return 'action-button';
         },
         /**
          * Passes `actionType` prop to action button if no `href` attribute is applied to the component
```

Comment:
Update comment with `no "href" or "to" attributes` to catch the logic below

Reply:
done